src/tests/ftest/erasurecode/aggregation.py

```diff
@@ -60,8 +60,8 @@ def test_ec_aggregation_default(self):
         :avocado: tags=ec_aggregation_default
         """
         self.pool.connect()
-        print("pool_percentage Before = {} ".format(
-            self.pool.pool_percentage_used()))
+        self.log.info("pool_percentage Before = {} ".format(
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions